README.md

```diff
@@ -291,7 +291,7 @@ backend {
 }
 ```
 
-Or, via `-Dbackend.abortJobsOnTerminate=true` command line option.
+Or, via `-Dserver.abortJobsOnTerminate=true` command line option.
```

Comment:
How about `system.abort-jobs-on-terminate`?  And yes I'm dashifying identifiers in here as long as I'm solving all the other problems.  :stuck_out_tongue: